apiary.apib

```diff
@@ -275,6 +276,7 @@ This collection is paginated.
               "releases": [{
                 "version": "0.4.1",
                 "url": "https://hex.pm/api/packages/plug/releases/0.4.1",
+                "html_url": "https://hex.pm/packages/plug/0.4.1",
```

Comment:
@ericmj I think it's fine, but just double-checking if it's ok to add it here, I know you were very concious about keeping `package.releases` payload as small as possible: https://hex.pm/api/packages/phoenix.